R/bcdc_browse.R

```diff
@@ -85,3 +85,34 @@ bcdc_browse <- function(query = NULL, browser = getOption("browser"),
 
 
 
+#' Generate a bibentry from a Data Catalogue Record
+#'
+#' Generate a bibentry object for use .bib file.
+#' for more details
+#'
+#' @inheritParams bcdc_get_record
+#' @export
+#'
+#' @seealso \link{utils::bibentry}
+#'
+#' @examples
+#'
+#' try(
+#' cite <- bcdc_get_data_citation("76b1b7a3-2112-4444-857a-afccf7b20da8")
+#'
+#' print(cite, stype = "citation")
+#' )
+bcdc_get_data_citation <- function(id) {
+  rec <- bcdc_get_record(id)
+
+  utils::bibentry(
+    bibtype = "Manual",
```

Comment:
Re adding "Province of British Columbia", can we use the "TechReport" `bibtype` which adds the "institution" field?

Reply:
Good suggestion. Here is where I landed for now
https://github.com/bcgov/bcdata/blob/9309444acaac97bca207316c0fc7f6c919d075d9/R/bcdc-get-citation.R#L78-L88

Open to additional input but we can also iterate more easily on this because it doesn't change actual code.

Reply:
Looks great to me!